gaek/ndb_json.py

```diff
@@ -120,9 +127,25 @@ def encode_basevalue(obj):
 }
 
 
+ALTERNATIVE_NDB_TYPE_ENCODING = {
+  ndb.Key: encode_key_as_pair,
+}
+
+
 class NdbEncoder(json.JSONEncoder):
   """Extend the JSON encoder to add support for NDB Models."""
 
+  # Whether to fetch entities specified by ndb.Key or to return them as pairs.
+  fetch_models=True
+
+  def __init__(self, skipkeys=False, ensure_ascii=True, check_circular=True, allow_nan=True, sort_keys=False,
```

Comment:
In order to future-proof this a bit, you can use `**kwargs` and then check for non-standard keyword arguments in the dictionary it provides.

Example:

```
def __init__(self, **kwargs):
  self._fetch_models = kwargs.pop('fetch_models', True)
  json.JSONEncoder.__init__(self, **kwargs)
```